.github/workflows/add-to-triage.yml

```
@@ -0,0 +1,16 @@
+name: Add to Triage
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
+        with:
+          project-url: https://github.com/orgs/eslint/projects/3
+          github-token: ${{ secrets.PROJECT_BOT_TOKEN }}
```

```suggestion
          github-token: ${{ secrets.PROJECT_BOT_TOKEN }}
          labeled: "triage:no"
          label-operator: NOT
```

To exclude issues with the `triage:no` label. For example, the scheduled release issues.

---

Good call!